core/templates/dev/head/pages/exploration_editor/EditorNavigationDirective.js

```diff
@@ -38,10 +38,14 @@ oppia.directive('editorNavigation', [function() {
         $scope.postTutorialHelpPopoverIsShown = false;
 
         $scope.$on('openPostTutorialHelpPopover', function() {
-          $scope.postTutorialHelpPopoverIsShown = true;
-          $timeout(function() {
+          if ($scope.isLargeScreen) {
```

Comment:
Actually -- I think everything gets initialized before anything runs, so the existing code is fine. Does it not trigger on your end?